### PR TITLE
Méli-mélo 2021-05-conjunction - Add support for no border

### PR DESCRIPTION
**Pull request by Christopher-O**
Added option to remove border to always stacked patterns when already contained in a visible design constraint (e.g. panel or zebra striping) to minimize the design and remove the redundant double border.  Also removed margin-left: 5px from stacked design to allow custom use of margin.

### CODE REVIEW

**Comment by duboisp** [comment]
Pre-approved upon successful completion of code review, build, testing and validating there is a working example showing that changes.

**Comment by Christopher-O** [comment]
> Pre-approved upon successful completion of code review, build, testing and validating there is a working example showing that changes.

My other PR is of an example for this update

**Comment by duboisp** [comment]
@Christopher-O I don't see your other PR, let connect.

File: méli-mélo/2021-05-conjunction/index.html [hunk 1]
```diff
@@ -100,10 +102,79 @@ <h4>Always stacked (mobile first, no side-by-side)</h4>
...
-&lt;/li&gt;</code>
+&lt;/li&gt;
+&lt;/ul&gt;</code>
   </pre>
   </details>
 
+<h3>No border on always stacked <span class="label label-info">New</span></h3>
+<p>This can be used when a border or design (e.g. zebra striping) has already defined the border of the content.</p>
+<h4>Always stacked (mobile first, no side-by-side)</h4>
+<div class="panel panel-default">
```
Comment (duboisp) [inline méli-mélo/2021-05-conjunction/index.html:113]: This structure need to be a sectioning element in order to have the "details/summary" content for the source code associated with the heading 4

File: méli-mélo/2021-05-conjunction/index.html [hunk 2]
```diff
@@ -100,10 +102,79 @@ <h4>Always stacked (mobile first, no side-by-side)</h4>
...
+        <ul class="cnjnctn-type-and brdr-0">
+    <li class="cnjnctn-col">This is content for part A </li>
+    <li class="cnjnctn-col">This is content for part B</li>
+</ul>
+	 </div>
+</div>
+
+   <details>
+    <summary>
+            <p>Code</p>
```
Comment (duboisp) [inline méli-mélo/2021-05-conjunction/index.html:138]: Remove the paragraph in the summary element

**Review by duboisp — CHANGES_REQUESTED** [review]
* [x] Content edit
* [x] Need a working example that show the no-border don't work with responsive conjunction

File: méli-mélo/2021-05-conjunction/index.html (hunk 1 above)
Comment (Christopher-O) [reply méli-mélo/2021-05-conjunction/index.html]: @duboisp  I have made the amendments, I have not yet created the test page for the options where the brdr-0 does not work, I can create that tomorrow.

**Review by duboisp — COMMENTED** [review]
Pre-approved upon review and when the missing working example (where the not CSS selector is applied) are demoed.

File: méli-mélo/2021-05-conjunction/index.html (hunk 1 above)
Comment (duboisp) [reply méli-mélo/2021-05-conjunction/index.html]: @Christopher-O I added the test page, we will be able to merge your PR now and include into our next deployment in AEM

**Review by duboisp — APPROVED** [review]
Change requested completed.

I reviewed the code and we have now an example for every possible configuration of this experimental feature.

**Pull request merged**